docs/source/getting_started/index.md

```diff
@@ -417,23 +415,24 @@ ragagent = Agent(
     ],
 )
 
-s_id = ragagent.create_session(session_name=f"s{uuid.uuid4().hex}")
+session_id = rag_agent.create_session(session_name=f"s{uuid.uuid4().hex}")
 
 turns = ["what is torchtune", "tell me about dora"]
 
 for t in turns:
     print("user>", t)
-    stream = ragagent.create_turn(
-        messages=[{"role": "user", "content": t}], session_id=s_id, stream=True
+    stream = rag_agent.create_turn(
+        messages=[{"role": "user", "content": t}], session_id=session_id, stream=True
     )
     for event in AgentEventLogger().log(stream):
         event.print()
 ```
-Run the script:
-```
-python rag_agent.py
+### ii. Run the Script
+Let's run the script using `uv`
+```bash
+uv run python rag_agent.py
 ```
-:::{dropdown} `Sample output`
+Which will output:
```

Comment:
This output is too long - it was dropdown because i didnt want the doc length to grow. Maybe we can have 3 tabs for each example - script, run, output? That way, we wont have to have a really long docs page.

Reply:
This output was only 10 lines, so I thought just displaying the output was fine. I'll switch it back to the dropdown 👍

I think the 3 tabs for script, run, output actually may be less intuitive for readers. As often readers will expect to move through examples sequentially (i.e., vertically) rather than horizontally.